fluentui_drawer/src/main/java/com/microsoft/fluentui/bottomsheet/BottomSheetAdapter.kt

```diff
@@ -78,17 +80,23 @@ class BottomSheetAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder> {
             listItemView.setTag(R.id.fluentui_bottom_sheet_item_divider, item.useDivider)
             listItemView.layoutDensity = ListItemView.LayoutDensity.COMPACT
             listItemView.background = R.drawable.bottom_sheet_item_ripple_background
+            listItemView.disabled = item.disabled
             if (textAppearance != 0) {
                 listItemView.titleStyleRes = textAppearance
             }
             if (subTextAppearance != 0) {
                 listItemView.subTitleStyleRes = subTextAppearance
             }
+
             if (item.customBitmap != null) {
-                listItemView.customView = context.createImageView(item.customBitmap)
+                image = context.createImageView(item.customBitmap)
             } else if (item.imageId != NO_ID) {
-                listItemView.customView = context.createImageView(item.imageId, item.getImageTint(context))
+                image = context.createImageView(item.imageId, item.getImageTint(context))
             }
+            if (image != null && item.disabled)
+                image?.imageAlpha = R.attr.fluentuiBottomSheetDisabledIconColor
```

Comment:
It's better to use attributes by using the correct theme from the context. Typecast to Fluent theme, please check the usage at other places

Reply:
Made changes as discussed